app/src/test/java/com/arctouch/codechallenge/HomeViewModelTest.java

```diff
@@ -0,0 +1,94 @@
+package com.arctouch.codechallenge;
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule;
+
+import com.arctouch.codechallenge.api.TmdbApi;
+import com.arctouch.codechallenge.api.TmdbApiSettings;
+import com.arctouch.codechallenge.data.Cache;
+import com.arctouch.codechallenge.home.HomeViewModel;
+import com.arctouch.codechallenge.model.Genre;
+import com.arctouch.codechallenge.model.Movie;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by eduardocucharro on 25/03/18.
+ */
+
+@RunWith(JUnit4.class)
+public class HomeViewModelTest {
```

Comment:
these tests are not working, do you think to do something with them?

Reply:
Yes, I should fix it. I did them before adding the search. Again, the rush, sorry! I planned to add some instrumentation tests too. But I thought it could be better to show you something before fix them!